### ISSUE TITLE
Flowchart resetting due to app.diagram.net not working properly

### ISSUE THREAD

**Issue by MxPie200** [algenty/grafana-flowcharting#406]
I'm still having the same issue as @bijwaard , in 2023.
I can open the drawing and its showing the mentioned warning:
![grafik](https://user-images.githubusercontent.com/81758431/233836596-a7c9f312-f58e-4d32-af03-e44f2c3255bc.png)

But if i make any changes to the Flowchart, the whole thing disappeared after closing the editor:

![grafik](https://user-images.githubusercontent.com/81758431/233836580-eb6d49fc-f331-4251-94a1-2edee971ede4.png)

How can I edit my drawing / how can we move to embed.diagrams.net ?
Changing the url in the "Source Content" does not work:
![grafik](https://user-images.githubusercontent.com/81758431/233836733-be605234-463e-4689-b68e-4ff2afb2cbdd.png)

I really like the Flowchart extension and would be great to hear about a fix for my issue.

_Originally posted by @MxPie200 in https://github.com/algenty/grafana-flowcharting/issues/230#issuecomment-1519039570_

**Comment by bijwaard** [comment]
Hi @MxPie200,
Have you tried to specify editor? This works for me!
![image](https://user-images.githubusercontent.com/6523124/233838015-4c1e9721-9264-4968-be2c-0d1d5a3e3df1.png)
Kind regards,
Dennis

**Comment by MxPie200** [comment]
@bijwaard 
Yes, I've tried it.
It opens the editor correctly, but after clicking on "Save" the Flowchart disappears as shown in my second screenshot

**Comment by bijwaard** [comment]
Aha, just tried editing my drawing, set it to light theme, and get the following back:

![image](https://user-images.githubusercontent.com/6523124/233838518-37f8e855-a064-49c0-8bc4-992e5c9f6c1c.png)

Looks like it reverts to the default diagram for some reason.

**Comment by MxPie200** [comment]
Hm ok, so it looks like it's a new bug with currently no solution for?

**Comment by bijwaard** [comment]
Looks like a new bug indeed. What version of the plugin are you using? I'm using 1.0.0d (on grafana 9.3.2).

```
$ sudo grafana-cli plugins ls
installed plugins:
ae3e-plotly-panel @ 0.5.0
agenty-flowcharting-panel @ 1.0.0d
alexandra-trackmap-panel @ 2.3.0
marcuscalidus-svg-panel @ 0.3.4
natel-plotly-panel @ 0.0.7
simpod-json-datasource @ 0.3.0

```

**Comment by MxPie200** [comment]
@bijwaard 
I have Version 0.9.1. , but grafana doesn't show updates, can i update via commandline?

**Comment by bijwaard** [comment]
Yes, you can update via the command line. You move the old plugin away from /var/lib/grafana/plugins and unzip the latest the beta release in the /var/lib/grafana/plugins folder, then restart grafana. Depending on which version you had previously, you may need to rename the folder to the old name.

**Comment by MxPie200** [comment]
Which files do i have to copy from github?

**Comment by bijwaard** [comment]
just tried on another grafana instance with plugin version 1.0.0b on grafana 9.1.5, and here the editor keeps working, but after saving the result stays blank in the panel, but further editing still works.

**Comment by bijwaard** [comment]
Latest snapshot is [here](https://github.com/algenty/grafana-flowcharting/tree/master/built). Current and older snapshots are [here](https://algenty.github.io/flowcharting-repository/ARCHIVES.html).

Bug may be a regression in 1.0.0d, and possibly a change in how embed.diagrams.net works for older versions.

**Comment by bijwaard** [comment]
you can rename your ticket to say that embed.diagrams.net is not working properly....

**Comment by MxPie200** [comment]
hm ok, i've had same as you that changes lead to reset to default flowchart..

**Comment by bijwaard** [comment]
Hopefully Arnaud just has to update/refresh the embed.diagrams.net library in the plugin....

**Comment by bijwaard** [comment]
looks like the same problem reported in #395

**Comment by manojrammurthy** [comment]
i have created a react equivalent of the plugin https://github.com/manojrammurthy/grafanaflowchart have a look at it and please give stars :)
